contracts/TandaPay.sol

```diff
@@ -1,7 +1,21 @@
 pragma solidity ^0.4.21;
 
+// Interface copied from WIkipedia
```

Comment:
Just to be clear, this does add the ERC20 interface, but doesn't actually implement it into the code right? The code still looks like a big improvement, so I'm going to approve+merge it, but just want to be clear that we still would need to do a bunch more work to add ERC20 into the project.